draft-ietf-snac-simple.xml

```diff
@@ -675,12 +683,12 @@
       <t>
 	In the fourth case, the stub router MUST provide its own NAT64 service.</t>
       <t>
-	An additional complication is that there may be more than one stub router connecting the stub network to infrastructure.
-	In this case, it may be desirable to limit the number of stub routers providing NAT64 service, or it may be acceptable
-	for all stub routers to provide it. In the latter case, this should not be a problem: since each stub router is using
-	its own ULA prefix to provide NAT64, any 5-tuple that goes through a stub router's NAT64 translator will necessarily
-	have as its destination an IPv6 address in a particular NAT64 prefix, and that address will select the correct stub
-	router through which to send the packet for translation.</t>
+	An additional complication is that there may be more than one stub router connecting the stub network to infrastructure.  In
+	this case, it may be desirable to limit the number of stub routers providing NAT64 service, or it may be acceptable for all
+	stub routers to provide it. In the latter case, this should not be a problem: since each stub router is using its own ULA
+	Site Prefix to provide NAT64, any 5-tuple that goes through a stub router's NAT64 translator will necessarily have as its
+	destination an IPv6 address in a particular NAT64 prefix, and that address will select the correct stub router through which
+	to send the packet for translation.</t>
```

Comment:
Possibly worth mentioning that this also works on the return path because each stub router has its own IPv4 address, and the return packet will be destined for that IPv4 packet, and hence will always return through the stub router that translated it on the way out.

Reply:
Updated.